app/src/main/java/com/poohcom1/playalong/ui/components/VideoPlayer.kt

```diff
@@ -15,7 +13,7 @@ import kotlinx.coroutines.delay
 import kotlinx.coroutines.withContext
 
 @Composable
-fun VideoPlayer(url: String, range: LongRange, tempo: Float) {
+fun VideoPlayer(url: String, range: LongRange, tempo: Float, currentPosition: MutableState<Long>) {
```

Comment:
You don't need to wrap states in `MutableState`. That is only needed when creating a new state. You can make this parameter a regular type like the other parameters